yapapi/rest/activity.py

```diff
@@ -89,8 +97,11 @@ async def __aexit__(self, exc_type, exc_val, exc_tb) -> None:
             with contextlib.suppress(yexc.ApiException):
                 # wait 1sec before kill
                 await self._api.get_exec_batch_results(self._id, batch_id, timeout=1.0)
-        except yexc.ApiException:
-            _log.error("failed to destroy activity: %s", self._id, exc_info=True)
+        except yexc.ApiException as err:
+            # Suppress errors that say that this activity does not exist (we're closing it anyway).
+            msg = f"No service registered under given address '/public/exeunit/{self._id}/Exec'"
```

Comment:
nitpicking but having `msg` here as the variable suggested to me that this is the message we'll _output_ to the logs and not one we were looking for in the error's body ... 

maybe it could be `silence(d)_msg` ? ...

Reply:
Fixed in https://github.com/golemfactory/yapapi/pull/122/commits/6450cbccec84d59d6a1736d99f11bd8d89a7f537